charts/argo-rollouts/values.yaml

```diff
@@ -482,8 +482,8 @@ notifications:
 
   # -- The subscriptions define the subscriptions to the triggers in a general way for all rollouts
   subscriptions: {}
```

Comment:
Though I should've noticed in previous PR, I wonder this should be list instead of object as following Argo CD's one. 🤔 
https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/values.yaml#L3369C3-L3369C16

Reply:
Also fixed.